src/NuGet.Clients/NuGet.CommandLine/Program.cs

```diff
@@ -134,7 +137,40 @@ public static int MainCore(string workingDirectory, string[] args)
                 else
                 {
                     SetConsoleInteractivity(console, command as Command);
-                    command.Execute();
+
+                    try
+                    {
+                        command.Execute();
+                    }
+                    catch (AggregateException e)
```

Comment:
This is new. 
What was happening in similar code paths before?

Reply:
This is to be able to throw the new `CommandLineArgumentCombinationException` and get the expected behavior. Depending on the path we want to follow I can wither keep this or remove it.